src/runtime/write_debug_image.cpp

```diff
@@ -114,6 +111,11 @@ WEAK uint8_t *get_pointer_to_data(int32_t dim0, int32_t dim1, int32_t dim2, int3
 
 }}} // namespace Halide::Runtime::Internal
 
+// We need a helper function that converts the return value to bool.
```

Comment:
Not sure if static linkage works OK in the runtime. Stick to alwaysinline or WEAK.